portfolio/src/main/webapp/index.html

```diff
@@ -47,6 +47,15 @@ <h1 class="about-text">About</h1>
         Women's Ultimate Frisbee Team and am a TA for CIS110: Introduction to Programming.   
       </p>
     </div> 
+
+    <div class="project">
+      <h1 id="project">Maps API</h1>
+      <div class="divider"></div>
+      <script type="text/javascript" src="https://www.gstatic.com/charts/loader.js"></script>
```

Comment:
Better to move script tags to the end of the body OR to the end of the header deferring them.

That way we don't block displaying any DOM below the script tag while it executes.